papaya-core/src/main/java/pl/edu/agh/papaya/security/UserContext.java

```diff
@@ -12,6 +12,10 @@ public String getUserId() {
         return getUser().getId().toString();
     }
 
+    public Long getUserIdn() {
+        return getUser().getId();
+    }
```

Comment:
Tym się na razie posługiwaliśmy tym idkiem jako stringiem, bo kiedyś najprawdopodobniej nim będzie - jak dla mnie nie ma problemu, żeby był Long, skoro tak na razie mamy w bazie, ale @kjarosh miał wizję i być może trochę ją niszczysz

Reply:
Już jest wszędzie intem, mój PR to wprowadza.

Reply:
55?

Reply:
Powiedz tylko gdzie a poprawię :smile:

Reply:
Mogło mi mignąć w czyimś diffie, ale będę czujna

Reply:
pewnie po wprowadzeniu Twoich zmian się będzie konflikciło, więc poprawię